### ISSUE TITLE
Make learning rate rescale based on mini-batch size rather than # workers

### ISSUE THREAD

**Issue by kyledmiller** [mala-project/mala#422]
Currently, mala rescales learning rate based on # of workers which indirectly follows the linear scaling rule ([Goyal et al.](https://arxiv.org/pdf/1706.02677.pdf)) if mini batch size is variably calculated based on # of workers. But this isn't always the case as mini batch size can be fixed separately. I think we should scale LR based on the mini batch size instead to ensure we are following the linear scaling rule.

**Comment by RandomDefaultUser** [comment]
I am closing this issue for now, but feel free to reopen as we go into multi-GPU training.
